src/caret_analyze/plot/bokeh/pub_sub_info_interface.py

```diff
@@ -45,26 +45,35 @@ def show(
         ywheel_zoom: bool = True,
         full_legends: bool = False,
         export_path: Optional[str] = None,
-    ) -> None:
+        interactive: bool = True
```

Comment:
Could you modify the default value as False?
```suggestion
        interactive: bool = False
```

Reply:
@hsgwa 
I have modified the default value as False.
Could you please review it again?